sha256.c

```diff
@@ -220,22 +220,23 @@ static void sha256_transform(u32 *state, const u8 *input)
 	memset(W, 0, 16 * sizeof(u32));
```

Comment:
Perhaps these are also unnecessary?

Reply:
Oh - didn't spot that.  Yeah - that's all dead as well.  I'll update the patch.